### ISSUE TITLE
Misc: this script "generate_doc.sh" seems to no more working but not really understand what this script done 

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#5018]
## Description and expectations

I find this script `https://github.com/tracim/tracim/blob/develop/generate_doc.sh` in the repository but not really understand if we need to support this anymore. The script not work at present:
~~~
./generate_doc.sh: ligne 8: /home/philippe/repo/tracim/backend/env/bin/sphinx-quickstart: Aucun fichier ou dossier de ce type
./generate_doc.sh: ligne 9 : cd: source: Aucun fichier ou dossier de ce type
~~~

I can seen V2.5 in script so I think its an old script.
I can seen also `make latexpdf`, `make epub` and `make html` at end of script but not understand what is the result of the script.

### How to reproduce

Try to run ./generate_doc.sh

### Version information

- Tracim version: on develop
- Device / System: Ubuntu 20.04

## Screenshots

________________________________________________

**Comment by inkhey** [comment]
Yes, as explained here: https://github.com/tracim/tracim/issues/2564

> add a "generate_doc.sh" script allowing with some tweak to create documentation for backend plugin using sphinx. This script is more at a stage of draft/poc than production documentation and is not documented yet.

This is a kind of poc of documentation done with sphinx created at the same time we do implement plugin mecanism, the idea there was to have a proper documentation for plugin that contain both manual doc and autogenerated doc from source code allowing to know all the existing hooks for creating a plugin.

**Comment by inkhey** [comment]
> ## Description and expectations
> 
> I find this script `https://github.com/tracim/tracim/blob/develop/generate_doc.sh` in the repository but not really understand if we need to support this anymore. The script not work at present:

It work if you install sphinx (`pip install sphinx`), as this is "poc", the documentation about building this documentation is not provided.

**Comment by PhilippeAccorsi** [comment]
ok.
